bootstrap.py

```diff
@@ -977,6 +981,7 @@ def exec_service(service, command, failonerror=True):
         print "PARTITIONTABLE - %s" % options.partitiontable
         print "ORG - %s" % options.org
         print "ACTIVATIONKEY - %s" % options.activationkey
+        print "CONTENT RELEASE" % options.release
```

Comment:
should be ` print "CONTENT RELEASE - %s" % options.release`

Reply:
ugh, right :face_with_head_bandage: